src/main/java/com/stripe/param/PaymentIntentCreateParams.java

```diff
@@ -37,6 +43,21 @@
   @SerializedName("confirm")
   Boolean confirm;
 
+  /**
+   * One of `automatic` (default) or `manual`.
+   *
+   * <p>When the confirmation method is `automatic`, a PaymentIntent can be confirmed using a
```

Comment:
is the `<p>` okay in javadoc?

Reply:
Yup <p> is a standard tag to add a preceding empty line